trustregistry/crud.py

```diff
@@ -0,0 +1,86 @@
+from sqlalchemy.orm import Session
+
+from . import models
+from . import schemas
+
+
+def get_actors(db: Session, skip: int = 0, limit: int = 1000):
+    return db.query(models.Actor).offset(skip).limit(limit).all()
+
+
+def create_actor(db: Session, actor: schemas.Actor):
+    db_actor = db.query(models.Actor).filter(models.Actor.id == actor.id).one_or_none()
+    if db_actor is not None:
+        return None
+    db_actor = models.Actor(**actor.dict())
+    db.add(db_actor)
+    db.commit()
+    db.refresh(db_actor)
+    return db_actor
+
+
+def delete_actor(db: Session, actor_id: str):
+    db_actor = db.query(models.Actor).filter(models.Actor.id == actor_id).one_or_none()
+    if db_actor is None:
+        return None
+    db.delete(db_actor)
+    db.commit()
+    return db_actor
+
+
+def update_actor(db: Session, actor: schemas.Actor, actor_id=str):
+    db_actor = db.query(models.Actor).filter(models.Actor.id == actor_id).one_or_none()
```

Comment:
How would this work? In almost all languages I know this would you evaluate to False before being passed to the function. Is this some Python magic at work?

```
filter(models.Actor.id == actor_id)
```